[2024-12-10T19:10:25.869+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-10T19:10:25.922+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.prepare_data manual__2024-12-10T19:10:11.685118+00:00 [queued]>
[2024-12-10T19:10:25.958+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.prepare_data manual__2024-12-10T19:10:11.685118+00:00 [queued]>
[2024-12-10T19:10:25.960+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-10T19:10:26.111+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): prepare_data> on 2024-12-10 19:10:11.685118+00:00
[2024-12-10T19:10:26.226+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=299) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-10T19:10:26.216+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'ml_pipeline', 'prepare_data', 'manual__2024-12-10T19:10:11.685118+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/mlops.py', '--cfg-path', '/tmp/tmpcczsviag']
[2024-12-10T19:10:26.250+0000] {standard_task_runner.py:72} INFO - Started process 316 to run task
[2024-12-10T19:10:26.255+0000] {standard_task_runner.py:105} INFO - Job 74: Subtask prepare_data
[2024-12-10T19:10:26.654+0000] {task_command.py:467} INFO - Running <TaskInstance: ml_pipeline.prepare_data manual__2024-12-10T19:10:11.685118+00:00 [running]> on host 15e8047cfc54
[2024-12-10T19:10:27.136+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='prepare_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-10T19:10:11.685118+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-10T19:10:11.685118+00:00'
[2024-12-10T19:10:27.141+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-10T19:10:27.194+0000] {logging_mixin.py:190} INFO - -----Inside prepare_data component----
[2024-12-10T19:10:27.457+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-10T19:10:27.494+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-10T19:10:27.496+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=prepare_data, run_id=manual__2024-12-10T19:10:11.685118+00:00, execution_date=20241210T191011, start_date=20241210T191025, end_date=20241210T191027
[2024-12-10T19:10:27.655+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-10T19:10:27.757+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-10T19:10:27.782+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
